cmd/launch-n-watch/main.go

```diff
@@ -79,13 +77,8 @@ func main() {
 func launchApp() int {
 	// launch app with args - save pid
 	cmd := exec.Command(flag.Arg(0), flag.Args()[1:]...)
-	stdoutIn, _ := cmd.StdoutPipe()
-	stderrIn, _ := cmd.StderrPipe()
-
-	var stdoutBuf, stderrBuf bytes.Buffer
-	var errStdout, errStderr error
-	stdout := io.MultiWriter(os.Stdout, &stdoutBuf)
-	stderr := io.MultiWriter(os.Stderr, &stderrBuf)
+	cmd.Stdin = os.Stdin
+	cmd.Stdout = os.Stdout
```

Comment:
I just realized that originally you were copying `stdout` and `stderr`, so you need to do something similar here:

```go
cmd.Stdin = os.Stdin
cmd.Stdout = os.Stdout
cmd.Stderr = os.Stderr
```

Otherwise if the launcher process writes to its `stderr` it won't be copied to the `stderr` of the launcher.

Reply:
That was it!
Interesting the Prometheus outputs all the logs to stderr